tests/integration/spec/Benchmark.ts

```diff
@@ -0,0 +1,68 @@
+import * as assert from 'assert';
```

Comment:
@charliesantos Please convert the file name to full lower case.

Reply:
We've been using PascalCase for class files in this repo...